tools/nanos.sh

```diff
@@ -1,6 +1,6 @@
-#!/bin/sh
 # run from parent directory of foam2
 cd foam2/build/
 mvn dependency:build-classpath -Dmdep.outputFile=cp.txt;
+cp services ../..
```

Comment:
Do what nanos does where it does a find and concatenates all services files together.